### PR TITLE
use native error instead of github.com/pkg/errors

### PR DESCRIPTION
**Pull request by wangyysde**
Signed-off-by: wangyysde <net_use@bzhy.com>

#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

xref #103043

#### Special notes for your reviewer:
NONE
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: hack/conformance/check_conformance_test_requirements.go
Comment (BenTheElder) [inline hack/conformance/check_conformance_test_requirements.go:75]: nit: this should probably just be errors.New ? (the stdlib errors)
lots of unnecessary use of format string processing in this file 😅

Comment (wangyysde) [reply hack/conformance/check_conformance_test_requirements.go]: Thanks . I have used errors.New instead of fmt.Errorf.

File: hack/conformance/check_conformance_test_requirements.go
Comment (BenTheElder) [inline hack/conformance/check_conformance_test_requirements.go:51]: https://blog.golang.org/go1.13-errors suggests there should be `%s: %w` (space before %w)

Comment (wangyysde) [reply hack/conformance/check_conformance_test_requirements.go]: Thanks. I have fixed it.

**Comment by BenTheElder** [comment]
fixed the release note (click edit and take a look, you'll see what it needs to look like)
/retest

File: hack/conformance/check_conformance_test_requirements.go
Comment (BenTheElder) [inline hack/conformance/check_conformance_test_requirements.go:89]: the previous comment about `: %w` applies to this as well

File: hack/conformance/check_conformance_test_requirements.go
Comment (BenTheElder) [inline hack/conformance/check_conformance_test_requirements.go:116]: the previous comment about `: %w` applies to this as well

**Review by BenTheElder — APPROVED** [review]

**Comment by BenTheElder** [comment]
above comments should still be fixed, but approved conceptually. will lgtm after those are fixed.

File: hack/conformance/check_conformance_test_requirements.go
Comment (wangyysde) [reply hack/conformance/check_conformance_test_requirements.go:89]: Thanks . I have fixed it.

File: hack/conformance/check_conformance_test_requirements.go
Comment (wangyysde) [reply hack/conformance/check_conformance_test_requirements.go:116]: Thanks . I have fixed it.

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve
/hold cancel
Thanks!

**Pull request merged**
